tests/test_sr.py

```diff
@@ -819,6 +821,660 @@ def test_finding_site(self):
         assert len(item.ContentSequence) == 0
 
 
+class TestSourceImageForSegmentation(unittest.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        file_path = Path(__file__)
+        data_dir = file_path.parent.parent.joinpath('data')
+        self._src_dataset = dcmread(
+            str(data_dir.joinpath('test_files', 'ct_image.dcm'))
+        )
+        self._ref_frames = [0, 4, 5]
```

Comment:
Frame numbers shall start at 1:

```suggestion
        self._ref_frames = [1, 4, 5]
```

Reply:
Applies to the other test cases as well.

Reply:
Addressed in 4bea49d. Also added a test on frame number (previously there was no check)